### ISSUE TITLE
screen reader and keyboard support

### ISSUE THREAD

**Issue by Eagalon** [LostRuins/koboldcpp#1904]
I love koboldcpp but I run into some trouble when using it via screen reader or keyboard. The tkinter library is not screen reader accessible by default, therefore, I would like to request if you can switch to using wxpython instead? as it has builtin modules for accessibility.
a second issue is regarding the image download. when I select the generated image, it doesn't show any way to download it via keyboard, it uses buttons like delete, or other options but no download option. could you make it so that the download option is similarly visible like the other options?

thank you and please keep up the good work.

**Comment by LostRuins** [comment]
Hello! Regarding the tkinter UI, yeah it's unfortunate but rewriting the entire UI launcher would be quite challenging and time consuming. Have you tried using the command line to launch it instead? That should be quite accessible for screen readers. You simply run "koboldcpp.exe --help" to get the available flags, and you can launch the model with a command as simple as "koboldcpp.exe --model your_file_name.gguf --usecuda" , adding whatever flags as needed.

For the second issue about image download, can I check, are you using it via the KoboldAI Lite Web UI, or via the StableUI SDUI?

**Comment by Eagalon** [comment]
thank you for responding. to answer your query, I am using it via kobold lite web ui. the rest of the webui is very much accessible but only image downloading is not possible. I have the same issue with both lite.koboldai.net as well as the local kobold lite webui.

as for porting over to wxpython, you are correct, it would be a huge undertaking and will require huge amount of time to do so. I greatly appreciate your response though, and yes. I have been using it via command line previously. it is a bit clunky to use but its completely accessible, yes.

once more thank you so much for making such an amazing app, of all the local ai apps, I love koboldcpp the most, as its very easy and quite intuitive to use with lots and lots of cool features.  🥰

**Comment by LostRuins** [comment]
Hello, the lite web ui has been updated and should now allow downloads
